tests/foreman/cli/test_contentview.py

```diff
@@ -1353,9 +1433,38 @@ def test_positive_publish_rh_and_custom_content(self):
 
         @assert: Content view can be published
 
-        @caseautomation: notautomated
-
+        @CaseLevel: Integration
         """
+        # Enable RH repo
+        self.create_rhel_content()
+        # Create custom repo
+        new_repo = make_repository({
+            'product-id': make_product({
+                'organization-id': self.rhel_content_org['id']})['id']
+        })
+        # Sync custom repo
+        Repository.synchronize({'id': new_repo['id']})
+        # Create CV
+        new_cv = make_content_view({
+            'organization-id': self.rhel_content_org['id'],
+        })
+        # Associate repos with CV
+        ContentView.add_repository({
+            'id': new_cv['id'],
+            'repository-id': self.rhel_repo['id'],
+        })
+        ContentView.add_repository({
+            'id': new_cv['id'],
+            'repository-id': new_repo['id'],
+        })
+        # Publish a new version of CV
+        ContentView.publish({'id': new_cv['id']})
+        new_cv = ContentView.info({'id': new_cv['id']})
+        self.assertTrue(
+            {self.rhel_repo['name'], new_repo['name']}.issubset(
+                {repo['name'] for repo in new_cv['yum-repositories']})
+        )
+        self.assertEqual(new_cv['versions'][0]['version'], '1.0')
```

Comment:
Are we using filters here ? I think test case description says `publish` the RH & custom filtered contents